### PR TITLE
ui2: Password Reminder Modal

### PR DESCRIPTION
**Pull request by amanalili**
#### PR Requirements

If you have made **any** visual changes to the GUI. Make sure you have:
- [ ] Tested on iOS Tablet
- [ ] Tested on small Android

Screen Shot 2021-01-04 at 11 13 56 PM

### CODE REVIEW

**Review by paullinator — COMMENTED** [review]
I have not reviewed the code, but have some visual feedback. 

1. Text field is not aligned on the left and right with the other content. It has a smaller left and right margin. 
2. Please make the title bold and left justified. 
3. Make the "I forgot" button the SecondaryButton.

File: src/components/scenes/WalletListScene.js [hunk 1]
```diff
@@ -54,6 +57,16 @@ class WalletListComponent extends React.PureComponent<Props, State> {
     }
   }
 
+  openPasswordReminderModal = () => {
+    if (this.props.isPasswordReminderVisible) {
```
[the remark this answers is not in the document]
Comment (amanalili) [reply src/components/scenes/WalletListScene.js]: Fixed

File: src/components/scenes/WalletListScene.js [hunk 2]
```diff
@@ -160,7 +173,8 @@ export const WalletListScene = connect(
     return {
       activeWalletIds,
       exchangeRates: state.exchangeRates,
-      wallets: state.ui.wallets.byId
+      wallets: state.ui.wallets.byId,
+      isPasswordReminderVisible: state.ui.passwordReminder.needsPasswordCheck
```
[the remark this answers is not in the document]
Comment (amanalili) [reply src/components/scenes/WalletListScene.js]: Fixed

**Comment by amanalili** [comment]
Fixed the styles

File: src/components/services/PasswordReminderService.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+import type { PasswordReminder } from '../../types/types.js'
+
+type Props = {
+  setPasswordReminder: (passwordReminder: PasswordReminder) => void,
+  passwordReminder: PasswordReminder,
+  loginStatus: boolean | null
+}
+
+class PasswordReminderComponent extends React.PureComponent<Props> {
+  UNSAFE_componentWillReceiveProps(nextProps: Props) {
```
Comment (paullinator) [inline src/components/services/PasswordReminderService.js:18]: Do not use deprecated method

File: src/components/services/PasswordReminderService.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    if (nextProps.loginStatus && !isEqual(this.props.passwordReminder, nextProps.passwordReminder)) {
```
Comment (paullinator) [inline src/components/services/PasswordReminderService.js:19]: Do not use lodash routines. We are trying to deprecate it from the code base  Check the parameters manually

**Review by paullinator — APPROVED** [review]
Please address comments in another PR

**Pull request merged**